packages/@glimmer/runtime/lib/scanner.ts

```diff
@@ -52,6 +52,12 @@ export default class Scanner {
             decorateTopLevelElement(tagName, symbols, attrs, newStatements);
           }
           addFallback(statement, newStatements);
+        } else {
```

Comment:
I don't intend this code to live long like this, I'm just doing the minimum for the tests.  I want to abstract it more when we add better tests and try integrating.